time stamp

float64[] deg_total_target
float64[] deg_total_now